### PR TITLE
Update etiquette.md

### PR DESCRIPTION
**Pull request by cottage14**
Minor text edits for readability.

### CODE REVIEW

File: source/contributors/etiquette.md [hunk 1]
```diff
@@ -54,45 +51,45 @@ Review and try to reformulate to be as clear as possible.
...
+**Most communities at Apache consists of level-headed, reasonable people** who
+have a strong, vested interest in their Apache project. This interest, often
+passion, can be a source of tension, but it is also what unites
 people within the community. It is easy to forget the vast amount of
 agreement that exists, and get upset over relatively small disagreements.
-Ability to put that aside, or downplay the importance, will ensure a
+Ability to put that aside, or downplay the importance of disagreements, will help ensure a
 harmonious project.
 
 **Face-to-Face** is excellent way to eliminate disagreements, but that is often
```
Comment (clr-apache) [inline source/contributors/etiquette.md:80]: Perhaps mention Zoom for completeness.

File: source/contributors/etiquette.md [hunk 2]
```diff
@@ -54,45 +51,45 @@ Review and try to reformulate to be as clear as possible.
...
-not practical. Consider Skype or Google Hangout, just for the social aspect
+not practical. Consider talking things out on Skype, Google Hangout or the project's Slack channel, just for the social aspect
 of being part of this community. It should not be formal, and the
-invitation should go out to everyone, perhaps someone want to make a short
+invitation should go out to everyonek. Perhaps someone will want to make a short
```
Comment (clr-apache) [inline source/contributors/etiquette.md:83]: s/everyonek/everyone/

**Review by clr-apache — CHANGES_REQUESTED** [review]
Thanks for your edits.

File: source/contributors/etiquette.md [hunk 3]
```diff
@@ -54,45 +51,45 @@ Review and try to reformulate to be as clear as possible.
...
+might not be needed. Once we can add a face and voice to the words we read, and a general
 idea how that person is socially, we are much more capable to interact by
 email.
 
-**Project decisions must always be made on mailing lists**.  
+**Project decisions must always be made on email lists**.  
 It's great to have a synchronous online chat to work through some questions, but 
 be sure to bring the results of the chat - and any proposals for the 
-project community to consider - back to the mailing list.  People who 
+project community to consider - back to the email list.  People who
```
Comment (rbowen) [inline source/contributors/etiquette.md:92]: Here, too, my preference is that we stick with the traditional "mailing list" nomenclature.

**Comment by rbowen** [comment]
Thanks. I'm happy to merge this PR, either with reverting "email list" back to "mailing list", or I can just make that change after merging, as you prefer. Let me know, and I'll try to check back in a few days.

**Comment by ShaneCurcuru** [comment]
Sorry! This page has already had similar and more extensive changes made, so closing this PR as no longer relevant.

**Pull request closed without merging**